src/TestReporter/ApiClient.php

```diff
@@ -22,11 +22,11 @@ public function __construct()
     /**
      * Send the given JSON as a request to the CodeClimate Server
      *
-     * @param object $json JSON data
+     * @param Entity\JsonFile $json JSON data
      *
      * @return \stdClass Response object with (code, message, headers & body properties)
      */
-    public function send($json)
+    public function send(Entity\JsonFile $json)
```

Comment:
This method is only used by [`UploadCommand::execute($input, $output)`](https://github.com/codeclimate/php-test-reporter/blob/master/src/ConsoleCommands/UploadCommand.php#L54), and `$json` in that case references the return value of [`CoverageCollector::collectAsJson()`](https://github.com/codeclimate/php-test-reporter/blob/master/src/TestReporter/CoverageCollector.php#L54-L66), which in turn returns an instance of `Entity\JsonFile`.